contracts/Hub.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.7;
+
+import "./samples/AuthorizedAddresses.sol";
+
+/**
+ * @title Hub Contract that coordinates Storage, Fund and Minter contract.
+ * @dev The contract can mint new synthtic assets (erc20 token), check the collateral ratio of addresses
+ * and liquidate addresses if the collateral ratio falls below 150%
+ * @author The Everest team.
+ */
+
+contract Hub is AuthorizedAddresses {
+    // Contract addresses
+    ICollateralFunds public fundContract;
+    IStorage public storageContract;
+    IUniswapV3Factory public factory;
+
+    mapping(string => address) public tickersymbolToSynthAssetContractAddress;
+    mapping(string => address) public tickerSymbolToTradingPool;
+
+    address public USDCKovan = 0xe22da380ee6B445bb8273C81944ADEB6E8450422;
+    address public uniswapV3Factory = 0x1F98431c8aD98523631AE4a59f267346ea31F984;
+
+    function getSynthAssetContractAddress (string memory tickerSymbol) public view returns (address) {
+        return tickersymbolToSynthAssetContractAddress[tickerSymbol];
+    }
+
+    constructor() {
```

Comment:
Is this part mandatory?

Reply:
You mean it's unnecessary because the variables are all public now?

Reply:
Hmm no just wondering if we need this empty constructor declaration :)